client/executor/wasmtime/src/runtime.rs

```diff
@@ -504,6 +504,27 @@ pub struct Semantics {
 
 	/// The heap allocation strategy to use.
 	pub heap_alloc_strategy: HeapAllocStrategy,
+
+	/// Enables WASM Multi-Value proposal
+	pub wasm_multi_value: bool,
+
+	/// Enables WASM Bulk Memory Operations proposal
+	pub wasm_bulk_memory: bool,
+
+	/// Enables WASM Reference Types proposal
+	pub wasm_reference_types: bool,
+
+	/// Enables WASM Fixed-Width SIMD proposal
+	pub wasm_simd: bool,
+
+	/// Enables WASM Threads and Atomics proposal
+	pub wasm_threads: bool,
+
+	/// Enables WASM Multi-Memory proposal
+	pub wasm_multi_memory: bool,
+
+	/// Enables WASM Memory64 proposal
+	pub wasm_memory64: bool,
```

Comment:
I don't think we need all of those, and some of those are pretty much experimental and should never be enabled until the spec for them is finalized.

Please remove `wasm_threads`, `wasm_multi_memory` and `wasm_memory64`.